docs/self-managed/setup/deploy/amazon/amazon-eks/eksctl.md

```diff
@@ -291,152 +334,98 @@ For detailed examples, review the [documentation provided by AWS](https://docs.a
 
 ```shell
-aws ec2 authorize-security-group-ingress \
-  --group-id $GROUP_ID \
-  --protocol tcp \
-  --port 5432 \
-  --cidr $CIDR
-  # the CIDR range should be exactly the same value as in the `cluster.yaml`
+kubectl create namespace camunda
 ```
 
-6. Retrieve subnets of the VPC to create a database subnet group:
+### Check Existing StorageClasses
 
-```shell
-export SUBNET_IDS=$(aws ec2 describe-subnets \
-  --filter Name=vpc-id,Values=$VPC_ID \
-  --query "Subnets[?Tags[?Key=='aws:cloudformation:logical-id']|[?contains(Value, 'Private')]].SubnetId" \
-  --output text | expand -t 1)
-```
-
-7. The variable `SUBNET_IDS` contains the output values of the private subnets (the value should look like this: `subnet-0123456789 subnet-1234567890 subnet-9876543210`).
+To see the available **StorageClasses** in your Kubernetes cluster, use the following command:
 
-8. Create a database subnet group to associate PostgreSQL within the existing VPC:
-
-```shell
-aws rds create-db-subnet-group \
-    --db-subnet-group-name camunda-postgres \
-    --db-subnet-group-description "Subnet for Camunda PostgreSQL" \
-    --subnet-ids $(echo $SUBNET_IDS)
+```bash
+kubectl describe storageclass
 ```
 
-9. Create a PostgreSQL cluster within a private subnet of the VPC.
-
-For the latest Camunda-supported PostgreSQL engine version, check our [documentation](../../../../../reference/supported-environments.md#camunda-8-self-managed).
-
-```shell
-aws rds create-db-cluster \
-    --db-cluster-identifier $RDS_NAME \
-    --engine aurora-postgresql \
-    --engine-version $POSTGRESQL_VERSION \
-    --master-username $PG_USERNAME \
-    --master-user-password $PG_PASSWORD \
-    --vpc-security-group-ids $GROUP_ID \
-    --availability-zones $(echo $ZONES) \
-    --database-name $DEFAULT_DB_NAME \
-    --db-subnet-group-name camunda-postgres
-```
+This command will display a list of defined StorageClasses, including which one is set as default.
 
-More configuration options can be found in the [AWS documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/create-db-cluster.html).
+#### StorageClass
 
-10. Wait for the PostgreSQL cluster to be ready:
+To check if `gp3` is set as the default StorageClass, you can look for the annotation `storageclass.kubernetes.io/is-default-class: "true"` in the output of the previous command. If `gp3` is not installed or not set as default StorageClass, you need to install it and set it as default.
 
-```shell
-aws rds wait db-cluster-available \
-    --db-cluster-identifier $RDS_NAME
-```
+We recommend using **gp3** volumes with Camunda 8 (see [volume performance](./amazon-eks.md#volume-performance)).
+It may be necessary to create the StorageClass, as the default configuration only includes **gp2**. For detailed information, refer to the [AWS documentation](https://aws.amazon.com/ebs/general-purpose/).
 
-11. Create a database instance within the DB cluster.
+#### Create `gp3` StorageClass
```

Comment:
should we include a node here that it requires the ebs csi cluster addon?
Just in case someone modified the defaults or is using their own yaml following this guide.